Podfile

```diff
@@ -1,22 +1,26 @@
-platform :ios, 8.0
-inhibit_all_warnings!
-
-def pods
-  pod "ObjectiveGumbo", "~> 0.1"
-  pod "AFNetworking", "~> 2.0"
-  pod 'AFNetworkActivityLogger', '~> 2.0'
-  pod 'ACKategories', :git => 'https://github.com/AckeeCZ/ACKategories.git', :commit => 'cc8e9b6'
-  pod 'SVProgressHUD', '~> 1.0'
-
-end
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
 
 target 'ISICbalance' do
+  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for ISICbalance
+  pod 'SnapKit', '~> 4.0.0'
+  pod 'ACKLocalization', '~> 0.1'
+  pod 'SwiftGen', '~> 6.0'
+  pod 'Alamofire', '~> 4.0.0'
+  pod 'SwiftSoup'
```

Comment:
pod bez verze 😱

Reply:
Hlavně že jsi to psal na slacku 🤦‍♂️ Polepším se.